src/server-fastly.js

```diff
@@ -15,7 +15,7 @@ import * as system from "./system.js";
 import * as util from "./commons/util.js";
 
 addEventListener("fetch", (event) => {
-  return serveDoh(event);
+  return event.respondWith(serveDoh(event));
```

Comment:
Ay ay ay (:

Reply:
We should add a basic HTTP test for all these applications to use, it would have caught this silly mistake I made 🙈

Reply:
True...

Only if I had a dollar each time prod went down because no tests... ;)